pyts2/pipeline/base.py

```diff
@@ -27,15 +27,15 @@ def process_image(self, image):
             image = step.process_image(image)
         return image
 
-    def process_from(self, input, ncpus=1):
+    def process_from(self, input_data, ncpus=1):
```

Comment:
Why `input_data` not `input`? Is it just that input overrides `input()`? Also, I think input/output are used in many places, and we should ensure consistency across all of them.

Reply:
Also if we use input_data, would make sense to have output_data. I also feel that we should use input_stream/output_stream to make clear what sort/format of data

Reply:
And by "many places" above, I think some of the timestream classes use `input` and `output`. Once this is merged, I'll make sure that they remain consistent, don't worry about it yourself (mainly since i'm reorganising some of that code).

Reply:
Yes exactly that reason.

I agree with having input_stream/output_stream